sdk/cosmos/azure-cosmos/azure/cosmos/database.py

```diff
@@ -128,42 +133,35 @@ def read(self,
         :rtype: Dict[Str, Any]
         :raises ~azure.cosmos.exceptions.CosmosHttpResponseError: If the given database couldn't be retrieved.
         """
-        # TODO this helper function should be extracted from CosmosClient
-        from .cosmos_client import CosmosClient
-
-        database_link = CosmosClient._get_database_link(self)
+        database_link = _get_database_link(self)
         request_options = build_options(kwargs)
         response_hook = kwargs.pop('response_hook', None)
         if populate_query_metrics is not None:
             warnings.warn(
                 "the populate_query_metrics flag does not apply to this method and will be removed in the future",
                 UserWarning,
             )
-            request_options["populateQueryMetrics"] = populate_query_metrics
```

Comment:
There are a couple of places where even though we have the warning that this param isn't supported, we were still adding it to the request_options

Reply:
@simorenoh - can you confirm that this would have just been ignored on the service (or errored), and therefore removing this will have no impact?